src/load.ts

```diff
@@ -89,9 +140,28 @@ export default async function load(
     } else {
       // load remote
       const headers = new Headers();
+
+      // Add custom parsed HTTP headers
+      if (options.httpHeaders) {
+        const parsedHeaders = parseHttpHeaders(options.httpHeaders);
+        Object.keys(parsedHeaders).forEach((headerKey) => {
```

Comment:
What about using `Object.entries` here instead of `Object.keys`?

Reply:
I know that we can access the value using the entries method, but is there a preference of why we would do this?￼

Reply:
Completed

Reply:
There is only one preference why I've asked about this: we rarely use the classic `for (let i=0; i<…)` statement to iterate a list without making any index magic because we have better version of that: for in, for of, forEach. These modern versions are less verbose and more semantically clear. So same point for `Object.entries`.